databricks/koalas/tests/test_groupby.py

```diff
@@ -183,6 +183,27 @@ def test_aggregate_func_str_list(self):
             sorted_agg_pdf = pdf.groupby(('X', 'A')).agg(aggfunc).sort_index()
             self.assert_eq(sorted_agg_kdf, sorted_agg_pdf)
 
+    def test_aggregate_relabel(self):
+        # this is to test named aggregation in groupby
+        pdf = pd.DataFrame({"group": ['a', 'a', 'b', 'b'],
+                            "A": [0, 1, 2, 3],
+                            "B": [5, 6, 7, 8]})
+        kdf = koalas.from_pandas(pdf)
+
+        # this is only applied in version after 0.25.0
+        if pd.__version__ < "0.25.0":
+            return
```

Comment:
Could you use `@unittest.skipIf( ... )` instead?

Reply:
nice!